devicetypes/smartthings/aeotec-doorbell-siren-6.src/aeotec-doorbell-siren-6.groovy

```diff
@@ -66,10 +110,24 @@ def installed() {
 	sendEvent(name: "alarm", value: "off", isStateChange: true, displayed: false)
 	sendEvent(name: "chime", value: "off", isStateChange: true, displayed: false)
 	sendEvent(name: "tamper", value: "clear", isStateChange: true, displayed: false)
+    	soundControl(2, 30, 17) //adjust the tamper volume to be lower than default when initially paired.
 }
 
 def updated() {
 	initialize()
+	if (SirenDoorbellSend == true) //keep Z-Wave traffic low, requires bool button in setting to trigger.
+	{
+    		soundControl(SirenDoorbellEndpoint, SirenDoorbellVolume, SirenDoorbellSound)
+	}
+}
+
+def soundControl(Endpoint, Volume, Sound) {
+	if (Endpoint && Volume && Sound) {
+    		log.debug "soundControl($Endpoint, $Volume, $Sound)"
+        	encap(zwave.multiChannelV3.multiChannelCmdEncap(sourceEndPoint:0, destinationEndPoint:Endpoint, commandClass:121, command:5, parameter: [Volume,Sound]))
```

Comment:
I generally see command class values in hexadecimal.

Reply:
@cbaumler 

commandClass = 0x79 //Sound Switch Command Class
command = 0x05 //Configuration SET

Reply:
@Aeotec-ccheng he's just saying that our typical coding style is to write these params as hex in the code

Reply:
Also, isn't the intent of the `encap` method to handle this? I don't think you should be passing a multichannel encapsulated command to `encap`

Reply:
@greens - shouldn't be a problem to change these values to hex.

Reply:
@greens - i can change the method of how the multichannel cmd encap is handled, but it does need to be secure encapsulated.

I've modified private encap() to properly pass multichannel encapsulation, but if needed, I can move it to a different def/func.

Reply:
@cbaumler @greens - Changed from decimal to hex value use and tested.